tests/cross/src/test/scala/tests/pc/CompletionCaseSuite.scala

```diff
@@ -24,6 +24,7 @@ class CompletionCaseSuite extends BaseCompletionSuite {
       |}""".stripMargin,
     """|case None => scala
        |case Some(value) => scala
+       |case
```

Comment:
I think it already suggests None: 
![image](https://github.com/scalameta/metals/assets/72950514/adea7d89-afa3-4b8a-bb10-c28f850d2cdf)

Are you thinking about suggesting only not covered cases?

Reply:
I think we should only show a single `case` if nothing else is shown to the user. Otherwise it's not that useful.

> I think it already suggests None:

Do we have a test case for that? I thought this not working was the issue raised?

Reply:
> I thought this not working was the issue raised?

I believe the issue was raised because there was no `case` completions at all, but this happened because there was no extending classes, just objects with `unapply`, which we don't check (maybe we could, by looking through object in scope with unapply method and correct argument?). 

> Do we have a test case for that?

No, I will add it.

> I think we should only show a single case if nothing else is shown to the user.
Sure, we can do that